lib/node_modules/@stdlib/math/base/special/expf/src/main.c

```diff
@@ -0,0 +1,233 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*
+* ## Notice
+*
+* The following copyrights, licenses, and long comment were part of the original implementation available as part of{@link https://svnweb.freebsd.org/base/release/12.2.0/lib/msun/src/e_expf.c?view=markup}, The implementation follows the original, but has been modified for JavaScript.
+*
+* e_expf.c -- float version of e_exp.c.
+* Conversion to float by Ian Lance Taylor, Cygnus Support, ian@cygnus.com.
+* ====================================================
+* Copyright (C) 1993 by Sun Microsystems, Inc. All rights reserved.
+* Developed at SunPro, a Sun Microsystems, Inc. business.
+* Permission to use, copy, modify, and distribute this
+* software is freely granted, provided that this notice
+* is preserved.
+* ====================================================
+*/
+
+#include "stdlib/constants/float32/ninf.h"
+#include "stdlib/constants/float32/pinf.h"
+#include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/math/base/special/expf.h"
+#include "stdlib/math/base/special/ldexpf.h"
+#include "stdlib/math/base/special/truncf.h"
+#include <stdint.h>
+
+static const float LN2_HI = 6.9314575195e-01;
+static const float LN2_LO = 1.4286067653e-06;
+static const float LOG2_E = 1.4426950216e+00;
+static const float half[ 2 ] = { 0.5, -0.5 };
+static const float EXP_OVERFLOW = 8.8721679688e+01;
+static const float EXP_UNDERFLOW = -1.0397208405e+02;
+static const float NEARZERO = 1.0 / ( 1 << 14 );
+static const float NEG_NEARZERO = -1.0 / ( 1 << 14 );
+
+/* Begin auto-generated functions. The following functions are auto-generated. Do not edit directly. */
+
+// BEGIN: polyval_p
+
+/**
+* Evaluates a polynomial for single-precision.
+*
+* ## Notes
+*
+* -   The implementation uses [Horner's rule][horners-method] for efficient computation.
+*
+* [horners-method]: https://en.wikipedia.org/wiki/Horner%27s_method
+*
+* @param x    value at which to evaluate the polynomial
+* @return     evaluated polynomial
+*/
+static float polyval_p( const float x ) {
```

Comment:
This function does not appear to be auto-generated using the appropriate package script.

Reply:
i have updated the coefficients in evalpoly now it is getting generated